flang/lib/Lower/OpenMP.cpp

```diff
@@ -400,7 +429,7 @@ static void genOMP(Fortran::lower::AbstractConverter &converter,
       currentLocation, lowerBound, upperBound, step, privateClauseOperands,
       firstPrivateClauseOperands, lastPrivateClauseOperands, linearVars,
       linearStepVars, scheduleClauseOperand.dyn_cast_or_null<StringAttr>(),
-      scheduleChunkClauseOperand,
+      scheduleChunkClauseOperand, /* schedule_modifiers */ nullptr,
```

Comment:
consistency nit: preferred syntax is `/*Foo=*/` for these inline argument comment annotations.

Reply:
Modified, so no longer there.